lib/core/EventBus.js

```diff
@@ -131,8 +133,9 @@ export default function EventBus() {
   this.on('diagram.destroy', 1, this._destroy, this);
 }
 
-
 /**
+ * @overlord
```

Comment:
@nikku  I'm not familiar with this keyword and couldn't find it anywhere? isn't the correct one `@overload`?

Reply:
This keyword is the working version of TypeScript `@overload`, cf. [documentation](https://github.com/nikku/bio-dts?tab=readme-ov-file#features).

Reply:
If you want to use `@overlord`, then you'd need to use `bio-dts` to generate the types. 

We do it in core bpmn.io as our sources remain `.js` and we still want to have clean types.